modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -58,7 +58,7 @@ You can import files into your project, either separately or bundled together in
 
 == What to do next
```

Comment:
Use title-style capitalization for all titles and headings, except for row headings in tables:

     What to Do Next